solutions/aiven-iceberg-tutorial/README.md

```diff
@@ -72,210 +45,181 @@ Before starting, ensure you have:
-    style D fill:#bbf,stroke:#333,stroke-width:2px
-    style E fill:#bbf,stroke:#333,stroke-width:2px
-    style F fill:#bbf,stroke:#333,stroke-width:2px
-    style G fill:#bbf,stroke:#333,stroke-width:2px
-    style H fill:#bbf,stroke:#333,stroke-width:2px
-    style I fill:#bbf,stroke:#333,stroke-width:2px
-    style J fill:#bbf,stroke:#333,stroke-width:2px
-```
+#### Step 1: Configure AWS CLI
+1. Install the AWS CLI if you haven't already.
+2. Run the following command to configure your AWS credentials:
+   ```bash
+   aws configure
+   ```
+   You'll be prompted to enter your AWS Access Key ID, Secret Access Key, region, and output format. These credentials will be used by Terraform automatically.
+
+#### Required AWS User Permissions
 
-#### Step 1: Create or use AWS IAM User
-1. Create an AWS User or use an existing one
-2. Make sure the following policy is attached to the user (either create or use existing policy):
-   ```json
-   {
+Your AWS user must have the following permissions to run the Terraform configuration:
+```json
+{
     "Version": "2012-10-17",
     "Statement": [
         {
             "Effect": "Allow",
             "Action": [
-                "iam:CreatePolicy",
                 "iam:CreateRole",
-                "iam:AttachRolePolicy",
+                "iam:DeleteRole",
                 "iam:GetRole",
+                "iam:PutRolePolicy",
+                "iam:CreatePolicy",
+                "iam:DeleteRolePolicy",
+                "iam:PassRole",
+                "iam:ListRolePolicies",
+                "iam:ListAttachedRolePolicies",
+                "iam:TagRole",
+                "iam:CreatePolicy",
+                "iam:DeletePolicy",
                 "iam:GetPolicy",
-                "iam:ListAttachedRolePolicies"
+                "iam:GetPolicyVersion",
+                "iam:ListPolicyVersions",
+                "iam:AttachRolePolicy",
+                "iam:DetachRolePolicy",
+                "iam:ListInstanceProfilesForRole",
+                "iam:RemoveRoleFromInstanceProfile",
+                "iam:DeleteInstanceProfile"
             ],
             "Resource": [
-                "arn:aws:iam::<account-id>:role/*",
-                "arn:aws:iam::<account-id>:policy/*"
+                "arn:aws:iam::<account-id>:role/snowflake_s3_role",
+                "arn:aws:iam::<account-id>:policy/snowflake_s3_policy"
             ]
         },
         {
             "Effect": "Allow",
             "Action": [
                 "s3:CreateBucket",
-                "s3:ListBucket",
+                "s3:DeleteBucket",
                 "s3:GetBucketLocation",
+                "s3:ListBucket",
+                "s3:PutObject",
+                "s3:ListAllMyBuckets",
+                "s3:GetBucketAcl",
+                "s3:PutBucketAcl",
+                "s3:GetBucketPolicy",
                 "s3:PutBucketPolicy",
-                "s3:GetBucketPolicy"
+                "s3:DeleteBucketPolicy",
```

Comment:
there are some bucket policies that may not be necessary... Can we double check? 

Generally these are the required permissions : 
`    {
        "Sid": "S3Access",
        "Effect": "Allow",
        "Action": [
          "s3:GetObject",
          "s3:PutObject",
          "s3:DeleteObject",
          "s3:ListBucket",
          "s3:GetBucketLocation",
          "s3:AbortMultipartUpload",
          "s3:ListMultipartUploadParts",
          "s3:ListBucketMultipartUploads"
        ],
        "Resource": [
          "arn:aws:s3:::<your-bucket-name>",  // Replace with your bucket name
          "arn:aws:s3:::<your-bucket-name>/*" // Replace with your bucket name
        ]
      }`

Reply:
Reference : https://aiven.io/docs/products/kafka/kafka-connect/howto/aws-glue-rest-catalog#configure-aws-iam-permissions